### PR TITLE
Fixed issue 5:

### PR DESCRIPTION
**Pull request by 123a6bcw**
Added method to check MoveMethodRefactoring on equality, choosing refactoring disables all rows with refactorings with the same source method, rows with the same method invalids after refactoring.

Снимок экрана от 2019-10-07 20-22-03
Снимок экрана от 2019-10-07 20-22-13

### CODE REVIEW

File: src/main/java/ui/MoveMethodTableModel.java [hunk 1]
```diff
@@ -126,8 +132,40 @@ public int getRowCount() {
...
         fireTableCellUpdated(virtualRow, columnIndex);
+        fireTableDataChanged();
+    }
+
+    private void setValueAtRowIndex(boolean select, int rowIndex, boolean forceSelectInConflicts) {
+        if (!isActive[rowIndex]) {
+            return;
+        }
+
+        for (int i = 0; i < refactorings.size(); i++) {
```
Comment (onewhl) [inline src/main/java/ui/MoveMethodTableModel.java]: Now, it is hard to understand why we need it here. I suggest to extract code from 148 to 166 lines to a separate method and give it a clear name.

File: src/main/java/ui/MoveMethodTableModel.java [hunk 2]
```diff
@@ -126,8 +132,40 @@ public int getRowCount() {
 
     @Override
     public void setValueAt(Object value, int virtualRow, int columnIndex) {
-        isSelected[virtualRows.get(virtualRow)] = (Boolean) value;
+        final int rowIndex = virtualRows.get(virtualRow);
+        final boolean select = (Boolean) value;
```
Comment (onewhl) [inline src/main/java/ui/MoveMethodTableModel.java]: Maybe `selected` is more appropriate name?

Comment (123a6bcw) [reply src/main/java/ui/MoveMethodTableModel.java]: `isSelected` already used, so not it's `isRowSelected`.

File: src/main/java/ui/MoveMethodTableModel.java (hunk 1 above)
Comment (123a6bcw) [reply src/main/java/ui/MoveMethodTableModel.java]: Done.

File: src/main/java/ui/MoveMethodTableModel.java [hunk 3]
```diff
@@ -126,8 +132,53 @@ public int getRowCount() {
...
+            return;
+        }
+
+        isSelected[rowIndex] = isRowSelected;
+    }
+
+    private boolean updateConflictingRows(boolean isRowSelected, int rowIndex, boolean forceSelectInConflicts) {
+        boolean hasConflicts = false;
+
+        for(int i = 0; i < refactorings.size(); i++)  {
```
Comment (onewhl) [inline src/main/java/ui/MoveMethodTableModel.java]: You should fix formatting, please, press `Ctrl` + `Alt` + `L`.

**Pull request merged**